File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.cs
public class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        long start = 0; long end = 0;
        if(a < b)
        {
            start = a;
            end = b;
        }
        else
        {
            start = b;
            end = a;
        }
        for(long i = start; i <= end; i++)
        {
            answer += i;
        }
        return answer;
    }
}

File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.cpp
#include <string>
#include <vector>

using namespace std;

long long solution(int a, int b) {
    long long answer = 0;
    if(a < b)
    {
        for(long long i = a; i <= b; i++)
        {
            answer += i;
        }  
    }
    else
    {
        for(long long i = a; i >= b; i--)
        {
            answer += i;
        }
    }
    
    return answer;
}